server/src/commandLine.mjs

```diff
@@ -89,6 +90,13 @@ const developerToolName = parsed.developerToolName || config.app.developerToolNa
 const proxy = parsed.proxy;
 const conduit = parsed.conduit;
 
+// Overriding multiUserConnections, if a configurartion, eg: allow/warn/deny (case-insensitive) is passed via CLI on starting MF server
+// Else default configuration 'warn' is used
+const multiUserConnections = parsed.multiUserConnections;
+if(/allow/i.test(multiUserConnections) || /warn/i.test(multiUserConnections) || /deny/i.test(multiUserConnections) == true){
```

Comment:
"== true" is usually a strange formulation in JS given how it treats 'truthiness'. It's okay, but you probably can just remove it if I understand what you're trying to do.

Reply:
Addressed